sources/packages/web/src/types/contracts/GeneralStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum GeneralStatus {
```

Comment:
Just a suggestion. I got confused on reading what is GeneralStatus. Can it be renamed like, ActiveStatus or anything more appropriate.

Reply:
It is  for all `generalStatus`, we can sync up, I can explain more